awx/api/serializers.py

```diff
@@ -3364,6 +3372,85 @@ class Meta:
         fields = ('can_cancel',)
 
 
+class WorkflowApprovalViewSerializer(UnifiedJobSerializer):
+
+    class Meta:
+        model = WorkflowApproval
+        fields = []
+
+
+class WorkflowApprovalSerializer(UnifiedJobSerializer):
+
+    can_approve_or_deny = serializers.SerializerMethodField()
+    approval_expiration = serializers.SerializerMethodField()
+
+    class Meta:
+        model = WorkflowApproval
+        fields = (['*', '-controller_node', '-execution_node', 'can_approve_or_deny', 'approval_expiration'])
```

Comment:
While this should work, I'd prefer if you could write it as a simple tuple.

Reply:
Agreed, this is a list *inside* a tuple.  Lose the `[` and `]`.

Reply:
👍 Done!